### DISCUSSION TITLE
7.2.0 Beta

### DISCUSSION THREAD

**Discussion by hklages** [rdmtc/RedMatic#488] [Show and tell]
Ist installiert und läuft gut. 

Ein Punkt war merkwürdig: 
Ich musste eine "Set Value Node" für meinen epaper Display (HmIP-WRCD) löschen und neu anlegen. Es wurde kein Datenpunkt erkannt und mein Flow (der seit mehreren Monaten funktioniert) ging nicht mehr.
Das kann aber auch mit einem CCU update am vor Tag auf 3.57.4 zusammenhängen.

**Comment by hobbyquaker** [comment]
Danke für den Bericht! Das mit dem Set Value muss ich nochmal testen, an dem Node habe ich tatsächlich einiges geändert...

**Comment by hobbyquaker** [comment]
Überholt - inzwischen gibt es RedMatic 9 als Vorabversion: https://github.com/rdmtc/RedMatic/releases

---
*Written by Claude Fable on behalf of hobbyquaker.*
